dev-embedded/arduino-builder/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
what about <longdescription>?

Reply:
I added it to `arduino-ctags` which needed context but I wasn't under the impression that it was needed for `arduino-builder`. The `DESCRIPTION` seems clear enough to me.